.eslintrc.js

```diff
@@ -2,5 +2,8 @@ module.exports = {
   extends: 'eslint-config-divisio',
```

Comment:
@luizbaldi what do you think about also adding `flow` and `jest`? not sure if it's appropriate or what the pros/cons might be

Reply:
we like both `flow` and `jest`! 

the point is, `flow` makes sense when you have an application that has data manipulation, so we use it to improve the typing structure, and `jest` is awesome for tdd.

since this is only a landing page, I see both of these technologies as overkill, and in the future, we can use them when needed :)

what do you think?

Reply:
yup sounds good! im a bit of a JS/react n00b, so still learning all the ins and outs of each toolkit, etc. i've also heard es6+ has static checking built in? i could be wrong about that :(